mcpo/entrypoint.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/sh
+
+echo "Running uname -m..."
+ARCH="$(uname -m)"
+echo "Raw architecture output: $ARCH"
+
+case "$ARCH" in
+  x86_64) ARCH_TAG="amd64" ;;
+  aarch64|arm64) ARCH_TAG="arm64" ;;
+  *)
+    echo "Unsupported architecture: '$ARCH'"
+    exit 1
+    ;;
+esac
+
+echo "Detected architecture tag: $ARCH_TAG"
+
+if [ -e "/var/run/docker.sock" ]; then
+  # Install Docker CLI
+  echo "Installing Docker CLI..."
+  curl -sSL "https://github.com/rancher-sandbox/rancher-desktop-docker-cli/releases/download/v28.1.1/docker-linux-${ARCH_TAG}" -o /usr/local/bin/docker
+  chmod +x /usr/local/bin/docker
+  echo "Docker CLI installed."
+
+  # Install Docker Compose
```

Comment:
I haven't checked if the MCP server supports it, but you may also want to install docker-buildx?

Reply:
No, the mcp server doesn't support docker-buildx